metsim/datetime.py

```diff
@@ -89,5 +90,7 @@ def units_from_freq(freq, origin=DEFAULT_ORIGIN):
         return 'hours since %s' % origin
     elif 'D' in freq:
         return 'days since %s' % origin
+    elif 'T' in freq:
+        return 'minutes since %s' % origin
     else:
-        raise NotImplementedError()
+        raise NotImplementedError('freq %s not supported at this time' % freq)
```

Comment:
I'd prefer we try to use `.format(...)` as much as possible.  It looks like there's a couple uses of the `%` formatting above as well that I didn't catch before.